addons/mail/static/src/models/partner/partner.js

```diff
@@ -195,6 +195,9 @@ function factory(dependencies) {
          * @returns {string}
          */
         _computeAvatarUrl() {
+            if (this === this.messagingPartnerRoot) {
```

Comment:
`this.env.messaging.partnerRoot` would be better I think, we usually don't reference "dependency fields" in compute methods.

Reply:
Indeed